lib/lambda_ethereum_consensus/fork_choice/fork_choice.ex

```diff
@@ -120,11 +83,10 @@ defmodule LambdaEthereumConsensus.ForkChoice do
       end
 
     persist_store(state)
-    {:noreply, state}
   end
 
-  @impl GenServer
-  def handle_cast({:attester_slashing, attester_slashing}, _state) do
+  @spec notify_attester_slashing(Types.AttesterSlashing.t()) :: :ok
+  def notify_attester_slashing(attester_slashing) do
```

Comment:
This is not a notification, as this is not a process. Let's call this `on_attester_slashing`. Also, instead of `state` we can call the variable `store`.

Also, let's only persist the store if it actually changes (the handler workded), not in the error case.

Reply:
That's right. Done!